cryptostock_api/cryptostock/celery_tasks/schedule_tasks.py

```diff
@@ -0,0 +1,138 @@
+import json
+
+from celery import shared_task
+from django.db import transaction
+from django.utils import timezone
+from django_celery_beat.models import IntervalSchedule, PeriodicTask
+from market.models import Market
+from notification.models import BrokerNotificationSubscription, BrokerNotificationType
+from utils.notification_handlers.common_services import notify
+from utils.notification_handlers.market_info_parser import MarketInfoParser
+
+
+def retrieve_assets_from_market():
+    for market in Market.objects.all():
+        market.kwargs["last_update_info"] = market.client.get_assets()
+        market.save()
+
+
+@shared_task(name="celery_tasks.schedule_tasks.scan_markets")
+def scan_markets():
+    return retrieve_assets_from_market()
```

Comment:
Minir, feel free to ignore
```suggestion
scan_markets = shared_task(name="celery_tasks.schedule_tasks.scan_markets"(retrieve_assets_from_market)
```

Reply:
ok